docs/Troubleshooting.md

```diff
@@ -98,12 +98,12 @@ Again, there are a number of possible causes and solutions:
   [#53](https://github.com/nteract/hydrogen/issues/53) the details of your
   installation.
 
-# Hydrogen No kernel specs found Issue!
+# Setting environment variables on Windows
 
-----------
 ## problem:
 
 In windows if python is not added to environment variables, you will get this `No kernel specs found` problem.
+> Make sure that you have Jupyer installed, if not use `py -m pip install jupyer` to install it
```

Comment:
I think we should leave this line out, since it should be covered on the installation page, we want to focus on the environment variables issue here :smile:

Reply:
correct !
let me do it...